lib/AArch64Base.ml

```diff
@@ -2260,8 +2251,32 @@ module
     | I_NOP -> true
     | _ -> false
 
-  let is_overwritable = is_overwritable
-  and can_overwrite =  can_overwrite
+(* In line with Arm ARM B2.2.5 "Concurrent modification and execution of instructions",
+   this function returns true iff its argument is one of:
```

Comment:
I think this function does the contrary?
```suggestion
   this function returns false iff its argument is any of:
```

Reply:
Yes -- good catch, thanks!